hare/broker.go

```diff
@@ -55,7 +55,8 @@ type Broker struct {
 	limit          int // max number of simultaneous consensus processes
 	stop           context.CancelFunc
 	eventLoopQuit  chan struct{}
-	queueMessageWg *sync.WaitGroup
+	eventLoopWg    sync.WaitGroup
```

Comment:
i think eventLoopQuit is doing the same as eventLoopWg, what was the point of adding it?

Reply:
@dshulyak It can be removed, you're right. Thanks